learn2track/models/gru_regression.py

```diff
@@ -48,13 +50,10 @@ def __init__(self, volume_manager, input_size, hidden_sizes, output_size, use_pr
         if self.predict_offset:
             assert self.use_previous_direction  # Need previous direction to predict offset.
 
+        # Do not use dropout in output layer
```

Comment:
Why is the output layer in the `gru_msp` dropped (actually it's the last hidden layer that is dropped according to how dropout is implemented in `LayerDense`) whereas this output layer is not?